libraries/apollo-compiler/api/apollo-compiler.api

```diff
@@ -773,15 +773,6 @@ public final class com/apollographql/apollo3/compiler/introspection/Introspectio
 	public fun toString ()Ljava/lang/String;
 }
 
-public final class com/apollographql/apollo3/compiler/introspection/IntrospectionSchemaJsonAdapter : com/squareup/moshi/JsonAdapter {
-	public fun <init> (Lcom/squareup/moshi/Moshi;)V
-	public fun fromJson (Lcom/squareup/moshi/JsonReader;)Lcom/apollographql/apollo3/compiler/introspection/IntrospectionSchema;
-	public synthetic fun fromJson (Lcom/squareup/moshi/JsonReader;)Ljava/lang/Object;
-	public fun toJson (Lcom/squareup/moshi/JsonWriter;Lcom/apollographql/apollo3/compiler/introspection/IntrospectionSchema;)V
-	public synthetic fun toJson (Lcom/squareup/moshi/JsonWriter;Ljava/lang/Object;)V
-	public fun toString ()Ljava/lang/String;
-}
-
```

Comment:
That's binary breaking but I haven't found a way to add the `@Suppress("DEPRECATION_ERROR")` annotation to Moshi generated files.